src/gui/shutdownconfirmdlg.cpp

```diff
@@ -30,20 +30,22 @@
  * Contact : hammered999@gmail.com
  */
 
-#include <QVBoxLayout>
-#include <QHBoxLayout>
+#include "shutdownconfirmdlg.h"
+#include "ui_shutdownconfirmdlg.h"
```

Comment:
Also, I propose we replace the old
```c++11
#ifndef HEADER_NAME_H
#define HEADER_NAME_H
#endif
```
with
```c++11
#pragma once
```

Modern compilers can handle it, not sure about some exotic platforms.
what do you guys think?